blender/arm/make_renderpath.py

```diff
@@ -27,9 +27,11 @@ def add_world_defs():
     # Screen-space ray-traced shadows
     if rpdat.arm_ssrs:
         wrd.world_defs += '_SSRS'
+        assets.add_khafile_def('rp_ssrs')
 
     if rpdat.arm_micro_shadowing:
         wrd.world_defs += '_MicroShadowing'
+        assets.add_khafile_def('rp_microshadowing')
```

Comment:
Why is this added?

Reply:
i really do not remember, might had something to do with one of my first pull request fixing translucency and/or tesselation with microshadowing.